qt/python/mantidqt/dialogs/errorreports/report.py

```diff
@@ -76,6 +76,9 @@ def __init__(self, parent=None, show_continue_terminate=False):
         # Dialog window to show more details of the crash to the user.
         self.details_dialog = MoreDetailsDialog(self)
 
+        # Set default focus to the editing box, rather then letting qt try and guess
+        self.input_free_text.setFocus()
```

Comment:
Thanks for fixing this. @DanielMurphy22 Can I just check that we didn't deliberately set the first field to have the cursor as the Name field for some reason?

Reply:
No I don't see any reason why we'd want that!